web/src/SidebarPin.tsx

```diff
@@ -49,18 +50,24 @@ export function SidebarPinContextProvider(
 
   function pinResource(name: string) {
     setPinnedResources(prevState => {
-      if (prevState.includes(name)) {
-        return prevState
-      } else {
-        return [...prevState, name]
-      }
+      const ret = prevState.includes(name) ? prevState : [...prevState, name]
+      incr("ui.web.pin", {
+        newPinCount: ret.length.toString(),
+        pinning: "true",
+      })
+      return ret
     })
   }
 
   function unpinResource(name: string) {
-    setPinnedResources(prevState =>
-      !prevState ? prevState : prevState.filter(n => n !== name)
-    )
+    setPinnedResources(prevState => {
+      const ret = !prevState ? prevState : prevState.filter(n => n !== name)
+      incr("ui.web.pin", {
+        newPinCount: ret.length.toString(),
+        pinning: "false",
```

Comment:
i'm not a huge fan of this but idk what else to change it to. i assume `true` -> it was a "pin" action and `false` -> it was an "unpin" action? this sorta reads to me as "anything is pinned: t/f". but also idk if there's a better name so eh.

Reply:
yeahhhh. I originally didn't have this tag and named the actions "ui.web.pin" and "ui.web.unpin", but I figured it'd be better for them to share a name to make querying for # of pinned items easier. Then I was going to go with `pinning: "true"` and `unpinning: "true"`, but that seems silly. `action: "pin"` and `action: "unpin"` seem a lot better, so I'll go with that. Thanks for prodding on this!